tensorboard/components/vz_line_chart/vz-line-chart.ts

```diff
@@ -107,6 +107,14 @@ Polymer({
      */
     xComponentsCreationMethod: {type: Object, value: () => stepX},
 
+    /**
+     * A formatter for values along the X-axis. Optional. Defaults to a
+     * reasonable formatter.
+     * 
+     * @type {@type {function(number): string}
```

Comment:
Please fix type signature.

Reply:
Thanks for the subtle catch! Also, I verified backwards compatibility.